tests/test_controller.py

```diff
@@ -164,21 +164,24 @@ def test_nearby(self):
             eq_("VENDORID", catalog["metadata"]["adobe_vendor_id"])
 
     def test_nearby_qa(self):
-        # The library we used in the previous test has stage=LIVE.
-        # If we switch to looking for libraries with stage=APPROVED,
-        # we won't find anything.
+        # The libraries we used in the previous test are in production.
+        # If we move them from production to TESTING, we won't find anything.
+        for library in self._db.query(Library):
+            library.registry_stage = Library.TESTING_STAGE
```

Comment:
In general, this code switched from "look for production libraries, then look for non-production libraries" to "look for production libraries, take them out of production, then verify they don't show up in the production feed, but they do show up in the QA feed."